templates/tonic-web-server-deployment.yaml

```diff
@@ -40,141 +39,36 @@ spec:
-          readOnlyRootFilesystem: true
-          {{- end }}
+          {{- include "tonic.web.container.securityContext" (list $ $server) | nindent 10 }}
         volumeMounts:
-        {{- if $server.volumeMounts }}
-        {{- toYaml $server.volumeMounts | nindent 10 }}
-        {{- end }}
-        {{- if .Values.readOnlyRootFilesystem }}
-        - name: var-log
-          mountPath: /var/log/supervisor
-        - name: var-run
-          mountPath: /var/run/supervisor
-        - name: tmp
-          mountPath: /tmp
-        - name: vector
-          mountPath: /tonic/vector_data
-        {{- end }}
+          {{- include "tonic.web.volumeMounts" (list $ $server) | nindent 10 }}
         env:
-        {{- range $key, $value := $server.env }}
-        - name: {{ $key }}
-          value: {{ $value | quote }}
-        {{- end }}
-        {{- if $server.envRaw }}
-        {{- with $server.envRaw }}
-            {{- toYaml . | nindent 12 }}
-        {{- end }}
-        {{- end }}
-        - name: TONIC_DB_DATABASE
-          value: {{ .Values.tonicdb.dbName }}
-        - name: TONIC_DB_USERNAME
-          value: {{ .Values.tonicdb.user }}
-        - name: TONIC_DB_PASSWORD
-          valueFrom:
-            secretKeyRef:
-              name: tonic-db-password
-              key: password
-        - name: TONIC_DB_PORT
-          value: {{quote .Values.tonicdb.port }}
-        - name: TONIC_DB_SSLMODE
-          value: {{ .Values.tonicdb.sslMode }}
-        - name: TONIC_DB_HOST
-          value: {{ .Values.tonicdb.host }}
-          {{- if .Values.tonicStatisticsSeed }}
-        - name: TONIC_STATISTICS_SEED
-          value: {{quote .Values.tonicStatisticsSeed }}
-          {{- end }}
-        - name: ENVIRONMENT_NAME
-          value: {{ .Values.environmentName }}
-        - name: TONIC_LICENSE
-          valueFrom:
-            secretKeyRef:
-              name: tonic-license-secret
-              key: license
-              optional: true
-        {{- if .Values.tonicSsoConfig }}
-        {{- include "tonic.sso" .Values.tonicSsoConfig | nindent 8 }}
-        {{- end }}
-        {{- if eq (include "tonic.hostIntegration" .) "true"}}
-        - name: TONIC_HOST_INTEGRATION
-          value: "kubernetes"
-        - name: TONIC_KUBERNETES_NAMESPACE
-          value: {{ .Release.Namespace }}
-        {{- end }}
-        {{- if $server.administrators }}
-        - name: TONIC_ADMINISTRATORS
-          value: {{ $server.administrators }}
-        {{- end }}
-        - name: TONIC_NOTIFICATIONS_URL
-          value: http://tonic-notifications:7000
-        {{- if not $httpsOnly }}
-        - name: TONIC_PORT_HTTP
-          value: {{ $httpPort | quote }}
-        {{- end }}
-        - name: TONIC_PORT_HTTPS
-          value: {{ $httpsPort | quote }}
-        - name: TONIC_HTTPS_ONLY
-          value: {{ $httpsOnly | quote }}
+          - name: TONIC_DISABLE_STARTUP_BACKGROUND_TASKS
+            value: {{ $separateBackgroundTasks | quote }}
```

Comment:
It might be safer to put this in an if block instead of setting the value at all.